tests/tlb/test_tlb_exception_fill.s

```diff
@@ -36,6 +36,8 @@
 #
 
 BEGIN_TEST_WITH_CUSTOM_TRAP_HANDLER
+        init_tlb
```

Comment:
These still need their indentation fixing (all but the one I commented on previously).